plugins/catalog-react/src/components/EntityRefLink/EntityRefLink.tsx

```diff
@@ -39,6 +60,101 @@ export type EntityRefLinkProps = {
   children?: React.ReactNode;
 } & Omit<LinkProps, 'to'>;
 
+type PeekAheadPopoverProps = {
+  popupState: PopupState;
+  entityRef: CompoundEntityRef;
+  ref: ForwardedRef<any>;
+};
+
+const useStyles = makeStyles(() => {
+  return {
+    popover: {
+      width: '80em',
```

Comment:
I would love advice on how I can get a consistent width for the popover. With these classes it looks wrong:

e.g.

![image](https://user-images.githubusercontent.com/553697/205599269-e00a3b9c-2abc-4618-92c2-56b69a1033f1.png)

and

![image](https://user-images.githubusercontent.com/553697/205599313-1308991d-aeec-48ac-8352-9f2ed4e0585c.png)